slidez-core/src/main/java/com/binarystudio/academy/slidez/domain/presentation/PresentationService.java

```diff
@@ -44,8 +45,14 @@ public Presentation assertPresentationExists(String presentationLink, String pre
         });
     }
 
-    public List getInfo() {
-        return this.presentationRepository.getInfo();
+    public List<PresentationDTO> getPresentationInfo() {
+        var presentations = this.presentationRepository.findAll();
+        ArrayList<PresentationDTO> presentationList = new ArrayList<>();
```

Comment:
Replace ArrayList with List on the left side of expression. Use actial types instead of "var"